holmes/checks/checks.py

```
@@ -0,0 +1,673 @@
+                slack_dest_config = destinations_config.get(dest_name, {})
+                webhook_url = (
+                    slack_dest_config.webhook_url if slack_dest_config else None
+                )
+
+                if webhook_url:
+                    # Use webhook URL for posting
+                    try:
+                        webhook_payload = self._format_slack_webhook_payload(
+                            check, result
+                        )
+                        response = requests.post(webhook_url, json=webhook_payload)
```

Ruff (0.14.13)

Probable use of `requests` call without timeout

(S113)